File: gateOne/DayTwo/Java Script/TaskTwo.js
let arrOfScores = [10];

for(let count = 0;count < 10;count++) {
	const prompt = require ("prompt-sync")();
	let score = parseInt(prompt("Enter Number: "));
	arrOfScores[count] = score;
}

for(let counter = 0;counter < 10;counter++) {

	console.log(arrOfScores[counter]);
}

File: gateOne/DayTwo/Java Code/TaskTwo.java
import java.util.Scanner;

public class TaskTwo {
 public static void main(String[] args) { 

	Scanner scanner = new Scanner(System.in);

	int[] arrOfScores = new int[10];

	for(int count = 0;count < 10;count++) {
		System.out.println("Enter Score");
		int scores = scanner.nextInt();
		arrOfScores[count] = scores;
	}

	System.out.println();

	for(int counter = 0;counter < 10;counter++) {

		System.out.println(arrOfScores[counter]);
	}




}

}